lib/rules/no-unused-vars.js

```diff
@@ -1152,7 +1152,7 @@ module.exports = {
 
 				// fix for { a: { b } }
 				if (parentNode.parent.type === "ObjectPattern") {
-					// fix for unused variables in dectructured object with single property in variable decalartion and function parameter
+					// fix for unused variables in dectructured object with single property in variable declaration and function parameter
```

Comment:
```suggestion
					// fix for unused variables in destructured object with single property in variable declaration and function parameter
```

Seems there is an another typo.